### PR TITLE
plugins/lsp: add openscad language server

### PR DESCRIPTION
**Pull request by anriha**
Just adds openscad lsp.

### CODE REVIEW

File: plugins/lsp/language-servers/default.nix [hunk 1]
```diff
@@ -410,6 +410,12 @@ let
         '';
       };
     }
+    {
+      name = "openscad-lsp";
+      description = "A Language Server Protocol server for OpenSCAD";
+      package = pkgs.openscad-lsp;
```
Comment (GaetanLepage) [inline plugins/lsp/language-servers/default.nix:416]:
```suggestion
```
Not needed.

Comment (MattSturgeon) [reply plugins/lsp/language-servers/default.nix:416]: (`mkLsp` defaults to using `pkgs.${name}`)

**Review by GaetanLepage — APPROVED** [review]
Thank you @anriha !

**Comment by MattSturgeon** [comment]
Can you squash your commits and use the commit message:

```
plugins/lsp/openscad-lsp: init
```

(See the LSP [file history](https://github.com/nix-community/nixvim/commits/c289738026ca31ef5b4875107c26ded4416d1697/plugins/lsp/language-servers/default.nix) for examples of previous commit messages)

**Comment by GaetanLepage** [comment]
@Mergifyio queue

**Comment by MattSturgeon** [comment]
@mergifyio dequeue

**Comment by GaetanLepage** [comment]
> Can you squash your commits and use the commit message:

Good catch

**Comment by MattSturgeon** [comment]
@mergifyio queue

**Pull request merged**